Source/JavaScriptCore/yarr/YarrErrorCode.cpp

```diff
@@ -65,6 +66,7 @@ ASCIILiteral errorMessage(ErrorCode error)
         REGEXP_ERROR_PREFIX "invalid property expression"_s,                          // InvalidUnicodePropertyExpression
         REGEXP_ERROR_PREFIX "too many nested disjunctions"_s,                         // TooManyDisjunctions
         REGEXP_ERROR_PREFIX "pattern exceeds string length limits"_s,                 // OffsetTooLarge
+        "Exhausted regular expression stack"_s,                                       // OutOfMemory
```

Comment:
The existing list is all syntax errors, either discovered during parsing or while converting to JIT code or ByteCompiling in the interpreter.

Please move this string down to the end and add a comment that this is a runtime error.  You'll need to update the enum as well.

Reply:
Fixed.  I've removed this `ErrorCode::OutOfMemory`.  Turns out to not be needed.